crates/search/src/buffer_search.rs

```diff
@@ -39,6 +41,7 @@ use registrar::{ForDeployed, ForDismissed, SearchActionsRegistrar, WithResults};
 
 const MIN_INPUT_WIDTH_REMS: f32 = 15.;
 const MAX_INPUT_WIDTH_REMS: f32 = 30.;
+const MAX_BUFFER_SEARCH_HISTORY_SIZE: usize = 20;
```

Comment:
I wonder if we should bump it too, to 100 or something, but no strong preference.

Reply:
I'll bump it to 50, that should be plenty, especially because its stored for each file